src/main/paradox/docs/additional-info/faq.md

```diff
@@ -1,3 +1,121 @@
+
+**What is JSON-LD?**
+
+JSON-LD is a JavaScript Object Notation for Linked Data. A JSON-LD payload is then converted to an RDF Graph for validation purposes and for ingestion in the Knowledge Graph.
+
+**What is RDF?**
+
+The Resource Description Framework (RDF) is a graph-based data model used for representing information in the Web. The basic structure of any expression in RDF is in triples, an extremely easy segmentation of any kind of knowledge in subject-predicate-object. It is a family of W3C specifications, and was originally design as a metadata model.
+ 
+**What is Elasticsearch?**
+
+Elasticsearch is a document oriented search engine with an HTTP web interface and schema-free JSON document. It’s able to aggregate data based on specific queries enabling the exploration of trends and pattern.
+
+**What is a SHACL schema?**
+
+SHACL (Shapes Constraint Language) is a language for validating RDF graphs against a set of conditions. These conditions are provided as shapes and other constructs expressed in the form of an RDF graph.
+SHACL is used in Nexus to constrain and control the payload that can be pushed into Nexus.
+
+
+**Do I need to define SHACL schemas to bring data in?**
+
+No. SHACL schemas provides with an extra layer of quality control for the data that is ingested into Nexus. However we acknowledge the complexity of defining schemas. That's why clients can decide whether to use schemas to constrain their data or not, depending on their use case and their available resources.
+
+
+**Where can I find SHACL shapes I can reuse (point to resources, like schema.org)?**
+
+In Neuroshape related [page on github](https://github.com/INCF/neuroshapes) are available shared validated data models based on open use cases.
+
+**Why are RDF and JSON-LD important for Nexus?**
+
+RDF is the data model used to ingest data into the Knowledge Graph and it is also used for SHACL schema data validation. JSON-LD is fully compatible with the RDF data model, and it is the main format we use for messages exchange. It is also the preferred format for message exchanges for APIs. The choice of JSON-LD is due to the fact that is as plain JSON but with some special [keywords](https://json-ld.org/spec/latest/json-ld/#syntax-tokens-and-keywords). 
+
+**Can I connect any SPARQL client to Nexus’ SPARQL endpoint?**
+
+Yes. As long as the client supports the ability to provide a ```Authentication``` HTTP Header (for authentication purposes) on the request, any SPARQL client should work.
+
+
+**How I can create a Organizations as an anonymous user in the docker-compose file? What needs to be done to switch to "authenticated" mode?**
+
+The permissions for anonymous are preset in the [ACLs](https://bluebrainnexus.io/docs/api/1.0/iam/iam-permissions-api.html) and should be replaced by the standard authentication. More details [here](https://bluebrainnexus.io/docs/api/1.0/iam/iam-permissions-api.html).
+
+
+**Can I use Nexus from Jupyter Notebooks?**
```

Comment:
Nexus can be used from Jupyter Notebooks using [Nexus Python SDK](https://github.com/BlueBrain/nexus-python-sdk/). Alternatively, you can also use any Python HTTP client and use Nexus REST API directly from the Jupyter Notebook.

Reply:
Maybe we can add some examples of jupyter notebooks that use nexus?
I think that is one: [Building a Data Pipeline for a Recommender System](https://github.com/BlueBrain/nexus/blob/master/src/main/paradox/docs/tutorial/notebooks/Recommendation%20System%20via%20Nexus.ipynb)

Reply:
I think these ones are also  [good examples](https://github.com/BlueBrain/nexus-python-sdk/tree/master/notebooks).